core/crypto/sr25519_signed.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_CRYPTO_SR25519_SIGNED_HPP
+#define KAGOME_CORE_CRYPTO_SR25519_SIGNED_HPP
+
+#include <type_traits>
+
+#include "common/blob.hpp"
+#include "crypto/sr25519_types.hpp"
+#include "scale/tie.hpp"
+
+namespace kagome::crypto::sr25519 {
+
+  using Signature = crypto::Sr25519Signature;
+  using SignerIndex = uint32_t;
+
+  template <typename D>
+  struct Sr25519Signed {
```

Comment:
Maybe move to "network/types/collator_messages.hpp"?

Reply:
Signed depends on Signature, which depends on algorithm. So it should locate near this algorithm and in our case can be either ed25519 or sr25519. So collator networking uses sr25519  and it has an alias to it.

Reply:
Is `Signed` used outside of collation protocol?